python/pysrc/kaskada/_timestream.py

```diff
@@ -199,6 +257,16 @@ def pipe(
         else:
             return func(self, *args, **kwargs)
 
+# TODO: How does udf work? We kind of want it like... apply some function on an input? 
```

Comment:
remove

Reply:
I think the udf should work as a function, so `my_udf(a, b)` should work. At which point, `a.pipe(my_udf, b)` should automatically work. You shouldn't need to change `pipe` to do that -- at that point, the `Udf` is a callable so it can be applied as the `func`, which `pipe` already does.